tutorials/22_PromptNode_with_Shaper.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>I really like this section. Shall we add them to the rest of our tutorials already @masci and @julian-risch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>